enterprise/security/src/test/java/org/neo4j/server/security/enterprise/auth/BoltProceduresIT.java

```diff
@@ -66,12 +69,13 @@ protected TestGraphDatabaseFactory getTestGraphDatabaseFactory()
     {
         return settings -> {
             settings.put( GraphDatabaseSettings.auth_enabled, "true" );
+            settings.put( GraphDatabaseSettings.plugin_dir, pluginsDirAbsPath() );
         };
     }
 
     @Override
     public NeoInteractionLevel<BoltInteraction.BoltSubject> setUpNeoServer() throws Throwable
     {
-        return new BoltInteraction( server );
+        return new BoltInteraction( new Neo4jWithSocket( getTestGraphDatabaseFactory(), getSettingsFunction() ) );
```

Comment:
Do we need to create a new server for every test? We cannot restart database when needed anymore?